app/controllers/spree/admin/products_controller_decorator.rb

```diff
@@ -33,6 +33,10 @@ def create
     delete_stock_params_and_set_after do
       super
     end
+  rescue Paperclip::Errors::NotIdentifiedByImageMagickError
+    invoke_callbacks(:create, :fails)
+    @object.errors.add(:base, t('spree.admin.products.image_upload_error'))
```

Comment:
may I suggest we let the user which formats we accept? I think this is the kind of messages that confuse the user "which format am I supposed to use?"

Reply:
tbh I have no clue what is that list...
I dont see this ImageMagick in our dependencies list... do you know if we use it?

Reply:
Maybe we can recommend common formats?

```diff
- Error uploading image, you may need to use a different image format
+ This image was not recognised. Please upload an image in PNG or JPG format.

Reply:
We install ImageMagick via ofn-install I think.

Reply:
nice one, I found this list: https://imagemagick.org/script/formats.php

Matt: that would make sense.

I'll follow Maikel's advise, looks reasonable.

Reply:
done